package.json

```diff
@@ -35,18 +32,19 @@
     "url": "git+https://github.com/jdiaz5513/capnp-ts.git"
   },
   "scripts": {
-    "bootstrap": "lerna bootstrap",
+    "bootstrap": "lerna bootstrap --hoist",
     "benchmark": "gulp benchmark",
     "build": "gulp build",
-    "ci": "gulp lint && gulp coverage && codecov --disable=gcov",
+    "ci": "gulp lint && gulp coverage && codecov --disable=gcov && cd js-examples && ./test.sh",
     "coverage": "gulp coverage",
+    "lerna": "lerna",
```

Comment:
nifty to have `npm run lerna -- ...` but let me now if you'd prefer not to have it.